.github/workflows/build-and-push-rag-content.yaml

```diff
@@ -0,0 +1,106 @@
+name: "Build and push rag-content container image"
+
+on:
+  workflow_dispatch:
+
+  pull_request:
+    branches:
+      - main
+    types:
+      - opened
+      - edited
+      - synchronize
+      - reopened
+
+  schedule:
+    - cron: '15 02 * * *'
```

Comment:
Period execution isn't necessary. We only need it on merging branches for publication of new changes and on new PRs for testing.

Please add: 
```
push:
    branches: [main]
```

or something similar.

Reply:
I disagree with this a bit. What if there is an update in the upstream documentation but no PRs into the openstack-lightspeed/rag-content?

Reply:
That would be better served by triggering this workflow on change to our documentation repos.

Reply:
I do not want to push on this too hard. Also, I do not think it is that important. Implementation of a trigger for this job when documentation gets updated in Opendev would be a little bit more involved. I think the schedule is a nice compromise. 

But if you insist on removing of `schedule` I'll surely do it:).

Reply:
It doesn't have to be done in this PR. We can give it a separate item.

Reply:
I'm marking this as resolved for now then. I added the push trigger.